vdirsyncer/vobject.py

```diff
@@ -40,7 +40,7 @@ class Item(object):
     VCARD'''
 
     def __init__(self, raw):
-        assert isinstance(raw, str)
+        assert isinstance(raw, str), type(raw)
```

Comment:
`type(raw)` is the error message. It's not related but should be there IMO either way.